Loop/View Controllers/BolusViewController.swift

```diff
@@ -9,24 +9,51 @@
 import UIKit
 import LocalAuthentication
 import LoopKit
+import LoopKitUI
 import HealthKit
 import LoopCore
+import LoopUI
 
 
-final class BolusViewController: UITableViewController, IdentifiableClass, UITextFieldDelegate {
+private extension RefreshContext {
+    static let all: Set<RefreshContext> = [.glucose, .targets]
+}
 
-    fileprivate enum Rows: Int, CaseCountable {
-        case notice = 0
-        case active
+final class BolusViewController: ChartsTableViewController, IdentifiableClass, UITextFieldDelegate {
+    private enum Row: Int {
+        case carbEntry = 0
+        case chart
+        case notice
         case recommended
         case entry
-        case deliver
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // This gets rid of the empty space at the top.
         tableView.tableHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: tableView.bounds.size.width, height: 0.01))
+
+        glucoseChart.glucoseDisplayRange = HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 60)...HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 200)
```

Comment:
Is this just a minimum display range? Or is this fixed? What if the actual prediction goes below 60 or above 200?

Reply:
This range is a placeholder for when there is no data / data hasn't yet loaded. The actual range is determined dynamically in the chart implementation.